src/deserialize_sequence.cc

```diff
@@ -358,6 +358,8 @@ void load_sequence_parameters(const std::filesystem::path& folder, Sequence& seq
 
             if (gul14::starts_with(keyword, "-- maintainers:"))
                 sequence.set_maintainers(keyword.substr(15));
+            else if (gul14::starts_with(keyword, "-- label:"))
+                sequence.set_label(gul14::trim(keyword.substr(9)));
```

Comment:
Between the maintainers and the label, it is a bit inconsistent where the `trim()` takes place (inside or outside the `set_` function).